### DISCUSSION TITLE
Background not visible?

### DISCUSSION THREAD

**Discussion by anirudhsama** [rxhanson/Charmstone-Community#10] [Q&A] [answered]
I downloaded the latest version from Charmstone from the Mac App Store and it looks like the background isn't visible. Is this a paid feature, or was this option removed?

**Comment by rxhanson** [comment, marked as answer]
The background isn't visible because it doesn't look very good having the empty space in the corner positions. There's a checkbox for it once the full version has been purchased, but you can enable it with this terminal command if you really want it:

`defaults write com.knollsoft.Charmstone background -int 1`

Then restart the app.

**Reply by anirudhsama** [reply]
Thanks!
